src/libs/actions/IOU.ts

```diff
@@ -675,7 +675,7 @@ function buildOnyxDataForMoneyRequest(
     ];
 
     // Policy won't be set for P2P cases for which we don't need to compute violations
```

Comment:
Sorry I missed this before, but this is now outdated
```suggestion
    // We don't need to compute violations unless we're on a paid policy
```